modules/launchd/default.nix

```
@@ -0,0 +1,192 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  inherit (pkgs.stdenv.hostPlatform) isDarwin;
+  inherit (lib.generators) toPlist;
+
+  cfg = config.launchd;
+  labelPrefix = "org.nixos.home.";
```

Maybe
```suggestion
  labelPrefix = "org.nix-community.home.";
```
since we're not an official nixos.org project.

---

I applied your suggestion and changed the tests accordingly. This made me realize there actually is a https://nix-community.org site.